java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -134,6 +135,7 @@ public static Object[] transactionTestResult() {
             2L,
             1L,
             1L,
+            true, // sismember(key7, "baz")
```

Comment:
```suggestion
            Boolean.TRUE, // sismember(key7, "baz")
```

Reply:
Not needed, it works as is

Reply:
we should update all tests if so, I think its fine to keep.